anitya/templates/project_new.html

```diff
@@ -109,7 +109,7 @@ <h1>{{ context }} project</h1>
 
         examples["{{ plugin.name }}"]="{{ plugin.examples | format_examples }}";
         more_info["{{ plugin.name }}"]="{{ plugin.more_info}}";
-        default_regex["{{ plugin.name }}"]="{{ plugin.default_regex }}";
+        default_regex["{{ plugin.name }}"]="{{ plugin.default_regex_html }}";
```

Comment:
So this could just be ``plugin.default_regex.replace('\\', '\\\\')``, I believe

I think this is fine, but just in case you hit some more complex (or user-provided) input that requires sanitizing, https://github.com/mozilla/bleach is a solid library used elsewhere in Fedora infra apps.